server/common/api/views.py

```diff
@@ -0,0 +1,93 @@
+from django.shortcuts import render
+
+# Create your views here.
+from django.http import HttpResponse, JsonResponse
+from api.models import * #import models
+
+#for csrf_exempt, not sure what is csrf
+from django.views.decorators.csrf import csrf_exempt
+
+from rest_framework import status, viewsets
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+
+#not know about what is sesrialization
+from api.serializers import UserSerializer
+
+
+
+import json
+
+def check_http_mehtod(request, type):
+	if request.type == type:
+		return True
+	else: 
+		return False
+
+
+
+
+def index(request):
+	return HttpResponse("Hello woRld! you're at the polls index.")
+
+
+@csrf_exempt
+def signup(request):
+	if request.method != 'POST':
+		return HttpResponse("Bad Request", status = 400)
+
+	t = json.loads(request.body)
```

Comment:
t is not a meaningful variable name. Every variable should have an easy to understand name.